oomox_gui/colors_list.py

```diff
@@ -316,10 +316,69 @@ def __init__(self, transient_for, callback):
         self.connect("clicked", self.on_click)
 
 
+class OomoxLinkedDropdown(Gtk.MenuButton):
+
+    drop_down = None
+
+    def build_dropdown_menu(self):
+        self.drop_down = Gtk.Menu()
+        menu_items = []
+        menu_items.append([Gtk.MenuItem(label="Replace all instances"), self.replace_all_instances])
+
+        for item in menu_items:
+            self.drop_down.append(item[0])
+            item[0].connect("activate", item[1])
+
+        self.drop_down.show_all()
+        return self.drop_down
+
+    def replace_all_instances(self, _menu_item):  # pylint:disable=unused-argument
+
+        _menu_item = self._menu_item  # noqa: F841
```

Comment:
however it doesn't seems you need this line at all, as `_menu_item` is not used next in the code